src/main/groovy/com/marklogic/gradle/task/roxy/RoxyGenerateTestScaffoldTask.groovy

```diff
@@ -0,0 +1,100 @@
+package com.marklogic.gradle.task.roxy
+
+import org.gradle.api.tasks.TaskAction
+
+//import com.marklogic.gradle.task.roxy.scaffoldTemplates.SampleTestFiles
+
+// Task mlGenerateRoxyTest : This task uses Groovy's SimpleTemplateEngine to customize the sample test suite
+// Parameters:
+//    roxySuitesDir   (Default: roxy/src/test/suites)
+//        - Specify the target directory for the sample test suite
+//    suiteName       (Default: SampleTestSuite)
+//        - Specify the name of the generated test suite
+//    testName        (Default: SampleTest)
+//        - Specify the name of the generated test
+// Examples:
+//    gradle mlGenerateRoxyTest
+//    gradle mlGenerateRoxyTest -PsuiteName="foo" -PtestName="bar" -ProxySuitesDir="./src/roxyTest/suites"
```

Comment:
Since this is an ml-gradle task, it should default to where ml-gradle expects to load modules from. I ran the task locally, and the modules were placed under roxy/src/test/suites. They should default instead to mlAppConfig.modulePaths, and I think the last value in that list (normally, there's only one path in that list, but an app can have multiple paths). You have access to this via the parent class - just call getAppConfig().getModulePaths()